lib/hackerone/client/incremental/activities.rb

```diff
@@ -0,0 +1,101 @@
+module HackerOne
+  module Client
+    module Incremental
+      class Activities
+        include ResourceHelper
+
+        DOTFILE = '.hackerone_client_incremental_activities'.freeze
```

Comment:
Is your plan to store the latest fetched thing in this file? If so, that isn't going to work if you have more than one machine that uses this lib and I would let this up to the user how to store this data.

Reply:
Yup, that's the plan! We need some kind of persistent storage to store the `updated_at_after` value, which essentially acts like a cursor.

I think you have a point, let's remove this logic?